critiquebrainz/frontend/external/musicbrainz_db/release_group.py

```diff
@@ -0,0 +1,94 @@
+from collections import defaultdict
+from brainzutils import cache
+from sqlalchemy.orm import joinedload
+from mbdata import models
+from critiquebrainz.frontend.external.musicbrainz_db import mb_session, DEFAULT_CACHE_EXPIRATION
+from critiquebrainz.frontend.external.musicbrainz_db.helpers import get_relationship_info, get_tags
+from critiquebrainz.frontend.external.musicbrainz_db.serialize import to_dict_release_groups
+from critiquebrainz.frontend.external.musicbrainz_db.includes import check_includes
+from critiquebrainz.frontend.external.musicbrainz_db.utils import get_entities_by_gids
+import critiquebrainz.frontend.external.relationships.release_group as release_group_rel
+
+
+def get_release_group_by_id(mbid):
+    """Get release group using the MusicBrainz ID."""
+    key = cache.gen_key(mbid)
+    release_group = cache.get(key)
+    if not release_group:
+        release_group = fetch_multiple_release_groups(
+            [mbid],
+            includes=['artists', 'releases', 'release-group-rels', 'url-rels', 'work-rels', 'tags']
+        )[mbid]
+    cache.set(key=key, val=release_group, time=DEFAULT_CACHE_EXPIRATION)
+    return release_group_rel.process(release_group)
+
+
+def fetch_multiple_release_groups(mbids, *, includes=None):
+    includes = [] if includes is None else includes
+    includes_data = defaultdict(dict)
+    check_includes('release_group', includes)
+    with mb_session() as db:
+        query = db.query(models.ReleaseGroup).options(joinedload("meta"))
```

Comment:
What does `.options(joinedload("meta"))` part do? It might be worth making a comment clarifying parts like these that might be more difficult to understand for someone unfamiliar with how mbdata works.